src/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+  </head>
+  <body>
+    <div class="container">
+      <!-- Header -->
+      <header>
+        <a href="">
+          <img
+            src="https://placehold.co/40x40/orange/white?font=Lora&text=YAS"
+            alt=""
+          >
+          <span>Your Amazing Shows</span>
+        </a>
```

Comment:
- [ ] Great work adding the number of movies, however, the requirements states that your project should look similar to the given design so kindly ensure the number of movies are placed as shown below:

### Given design
![Screenshot 2023-03-18 at 23 56 00 (2)](https://user-images.githubusercontent.com/64774936/226144410-1eeef9de-a805-430f-8a4c-fc864bc27d1d.png)

### Your design
![Screenshot 2023-03-18 at 23 55 51 (2)](https://user-images.githubusercontent.com/64774936/226144415-b177e09d-84d7-434e-8077-dddda8c98b69.png)